### DISCUSSION TITLE
validate session token in express server

### DISCUSSION THREAD

**Discussion by Shivam00123** [nextauthjs/next-auth#11679] [Help]
i have next-auth v5 running and it is working fine for frontend, now i am sending session token as a cookie to my express server and i want to authenticate the user, but i tried my level best but could not succeed decoding it on my express server. Is there a way to do it documentation is not helping much.

**Comment by 9021007** [comment]
Having the EXACT same issue. There doesn't appear to be a function that just checks the session token passed as a cookie. Very annoying.

**Comment by hillac** [comment]
Answered here: https://github.com/nextauthjs/next-auth/discussions/10955#discussioncomment-10321414
